layers/+source-control/git/README.org

```diff
@@ -194,16 +195,25 @@ intended. To work around this, you can use =v= instead (since Magit only stages
 whole lines, in any case).
 
 ** Commit message editing buffer
-In a commit message buffer press ~​,​c~ (if =dotspacemacs-major-mode-leader-key= is ~​,​~)
-or ~C-c C-c~ to commit the changes with the entered message. Pressing ~​,​a~ or ~C-c C-k~
-will discard the commit message.
-
-| Key binding | Description |
-|-------------+-------------|
-| ~h~         | go left     |
-| ~j~         | go down     |
-| ~k~         | go up       |
-| ~l~         | go right    |
+In a commit message buffer the following key bindings are active:
+
+| Key binding            | Description                                               |
+|------------------------+-----------------------------------------------------------|
+| ~SPC m c~ or ~SPC m ,~ | commit changes with the entered message                   |
+| ~SPC m a~ or ~SPC m k~ | discard message and abort the commit                      |
+| ~g j~                  | cycle through history to the previous commit message      |
+| ~g k~                  | save current commit message and cycle to the next message |
```

Comment:
In the commit edit buffer, I see that <kbd>g j</kbd> is bound to `evil-next-visual-line`, and <kbd>g k</kbd> is bound to `evil-previous-visual-line`.  Are they bound differently for you? I do see that <kbd>M-n</kbd> is `git-commit-next-message`, and <kbd>M-p</kbd> is `git-commit-prev-message`.

Reply:
I see that evil-magit binds <kbd>&lt;visual state&gt; g j</kbd> to `git-commit-next-message`, but <kbd>g j</kbd> is `evil-next-visual-line` and takes precedence for me in both the vim editing style and the hybrid editing style.

Reply:
That's interesting - I'm on vim editing style and `g j` / `g k` are bound to the commit cycling functions, whereas `M-n` and `M-p` are unbound.

Reply:
Can anyone else using Evil + Magit confirm if the `gj/gk` or `M-n/M-p` bindings work for them in the commit message editing buffer? 

(This isn't part of the original PR's scope but I thought I'd just lump it in, since they take so long to get merged around here)

Reply:
@yuhan0 I see the same thing as @Miciah in my setup.